perfmetrics/scripts/fio/constants.py

```diff
@@ -0,0 +1,128 @@
+from dataclasses import dataclass
+from typing import Any, Dict, List, Tuple, Callable
+
+GLOBAL_OPTS = 'global options'
+JOBS = 'jobs'
+JOB_OPTS = 'job options'
+PARAMS = 'params'
+FILESIZE = 'filesize'
+FILESIZE_KB = 'filesize_kb'
+NUMJOBS = 'numjobs'
+THREADS = 'num_threads'
+TIMESTAMP_MS = 'timestamp_ms'
+RUNTIME = 'runtime'
+RAMPTIME = 'ramp_time'
+STARTDELAY = 'startdelay'
+START_TIME = 'start_time'
+END_TIME = 'end_time'
+RW = 'rw'
+READ = 'read'
+WRITE = 'write'
+METRICS = 'metrics'
+IOPS = 'iops'
+BW_BYTES = 'bw_bytes'
+IO_BYTES = 'io_bytes'
+LAT_NS = 'lat_ns'
+MIN = 'min'
+MAX = 'max'
+MEAN = 'mean'
+PERCENTILE = 'percentile'
+P20 = '20.000000'
+P50 = '50.000000'
+P90 = '90.000000'
+P95 = '95.000000'
+
+NS_TO_S = 10**(-9)
+
+
+@dataclass(frozen=True)
```

Comment:
constants file should have just constants. please move JobParam, JobMetric and REQUIRED_JOB_PARAMS to fio_metrics.py class.
If you think fio_metrics.py is too big create seeperate classes for jobMetric and JobParam, required_job_params should still be in fio_metrics.py

Reply:
Done. Moved JobParam, JobMetric and REQUIRED_JOB_PARAMS to fio_metrics.py